### PR TITLE
More text edits, and added a ProductHunt banner.

### PR DESCRIPTION
**Pull request by AnnaPetrovicheva**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Review by nmanovic — COMMENTED** [review]
@AnnaPetrovicheva , could you please merge develop? I fixed many markdown linter issues. You need to take these changes.

**Comment by AnnaPetrovicheva** [comment]
Done - apologies. Need to change my local linter settings.

File: README.md [hunk 1]
```diff
@@ -177,26 +179,21 @@ The exact steps on how FFmpeg was configured and compiled can be found in the [D
...
-CVAT usage related questions or unclear concepts can be posted in our
-[Gitter chat](https://gitter.im/opencv-cvat) for **quick replies** from
-contributors and other users.
+## Where to ask questions
 
-However, if you have a feature request or a bug report that can reproduced,
-feel free to open an issue (with steps to reproduce the bug if it's a bug
-report) on [GitHub\* issues](https://github.com/cvat-ai/cvat/issues).
+[Gitter chat](https://gitter.im/opencv-cvat): you can post CVAT usage related there.
+Typically they get answered fast by the core team or community. You can also browse other users common questions on our gitter.
```
Comment (nmanovic) [inline README.md:189]:
```suggestion
Typically they get answered fast by the core team or community. You can also browse other users
common questions on our gitter.
```

File: README.md [hunk 2]
```diff
@@ -177,26 +179,22 @@ The exact steps on how FFmpeg was configured and compiled can be found in the [D
...
+Typically they get answered fast by the core team or community. You can also browse other users
+common questions on our gitter.
 
-If you are not sure or just want to browse other users common questions,
-[Gitter chat](https://gitter.im/opencv-cvat) is the way to go.
+[GitHub issues](https://github.com/cvat-ai/cvat/issues): please post your have feature requests or bug reports there.
+If it's a bug, please add the steps to reproduce it.
 
-Other ways to ask questions and get our support:
+[\#cvat](https://stackoverflow.com/search?q=%23cvat) tag on StackOverflow is one more way to ask questions and get our support.
```
Comment (nmanovic) [inline README.md:195]:
```suggestion
[\#cvat](https://stackoverflow.com/search?q=%23cvat) tag on StackOverflow is one more way to ask
questions and get our support.
```

**Review by nmanovic — APPROVED** [review]
@AnnaPetrovicheva , awesome!

**Pull request merged**